### ISSUE TITLE
Parameter "version" error of build.gradle file.

### ISSUE THREAD

**Issue by mathsigit** [SparkMonitor/varOne#45]
In branch 0.1.1, due to the parameter "version" in build.gradle file was not the same with the branch version number, there were no jars (varOne-server-{version}.jar and varOne-node-{version}.jar) in the varOne-tgz file.

Maybe we should set version number as input parameter.

**Comment by AllenFang** [comment]
``` sh
./bin/release.sh RELEASE_VERSION
```

Actually, make version in gradle configurable is better solution.

check https://github.com/SparkMonitor/varOne/blob/0.1.1/bin/release.sh#L24

**Comment by mathsigit** [comment]
Yep! That's what I mean, setting a parameter of version in gradle, and send it when running `./bin/release.sh RELEASE_VERSION`
